cache/src/main/scala/org/allenai/common/cache/JsonQueryCache.scala

```diff
@@ -25,7 +41,10 @@ class JsonQueryCache[V: JsonFormat] @Inject() (
     * become part of the cache key (prepended to the actual query)
     */
   def this(redisHostName: String, redisHostPort: Int, redisTimeout: Int, clientPrefix: String) =
-    this(new JedisPool(new JedisPoolConfig, redisHostName, redisHostPort, redisTimeout), clientPrefix)
+    this(
+      new JedisPool(new JedisPoolConfig, redisHostName, redisHostPort, redisTimeout),
```

Comment:
I really don't like multiple constructors on objects. It makes initialization a lot harder to get right. :/